build.gradle

```diff
@@ -10,14 +10,88 @@ plugins {
   id 'au.com.dius.pact' version '4.1.10'
   id "io.freefair.lombok" version "5.3.0"
   id "org.flywaydb.flyway" version "7.1.1"
+  id 'net.ltgt.apt' version '0.21'
 }
 
 group = 'uk.gov.hmcts.reform'
 version = '0.0.1'
 
 allprojects {
+
```

Comment:
lots of the changes here seem irrelevant to PR

Reply:
really 🤔  which ones?